src/api/collections-api.ts

```diff
@@ -2,13 +2,11 @@ import { getConfig, CollectionsApi, ListCollectionsResponse } from "@imtbl/core-
 
 // Get a list of collections
 // Inspired from Immutable Core SDK documentation
-const GetYourCollections = async (): Promise<ListCollectionsResponse> => {
+export const getCollections = async (): Promise<ListCollectionsResponse> => {
   const config = getConfig("ropsten");
```

Comment:
I believe the best way here would be to use the DotEnv lib to manage config based in the .env file